koin-projects/examples/multimodule-ktor/module-b/src/main/kotlin/org/koin/samples/multi_module/module_b/moduleB.kt

```diff
@@ -0,0 +1,28 @@
+package org.koin.samples.multi_module.module_b
+
+import io.ktor.application.Application
+import io.ktor.application.call
+import io.ktor.response.respondText
+import io.ktor.routing.get
+import io.ktor.routing.routing
+import org.koin.dsl.bind
+import org.koin.dsl.module
+import org.koin.experimental.builder.single
+import org.koin.ktor.ext.inject
+import org.koin.ktor.ext.koinModules
+import org.koin.samples.multi_module.common.IService
+
+fun Application.moduleB() {
+    koinModules(
```

Comment:
perhaps just `modules` instead of `koinModules` in fact :)

Reply:
In the process of writing, he wrote like this so as not to get confused. When committing, I didn’t fix it - now I fix it :)